app/models/pillar.rb

```diff
@@ -61,7 +62,13 @@ def apply(pillars, required_pillars: [], unprotected_pillars: [])
 
     private
 
+    # all pillars that are not in the Pillar database
+    def independent_pillars
+      [:insecure_registries]
```

Comment:
Are we gonna use `insecure_registries`? Because we'll handle insecure, secure and mirrors.